test/inductor/test_fp8.py

```diff
@@ -346,6 +368,67 @@ def ln_fp8(x: Tensor, scale: Tensor, amax_buffer: Tensor):
         )
 
 
+@instantiate_parametrized_tests
+class TestFP8Lowering(TestCase):
+    @unittest.skipIf(TEST_WITH_ROCM, "FP8 is not supported on ROCM")
```

Comment:
cc @drisspg , thoughts on how far away we are from also supporting the nuz float8 flavors here?